main.mm

```diff
@@ -7,33 +7,19 @@
 #import "cmdline.h"
 
 
-int main(int argc, const char* argv[])
+int main(int argc, char* argv[])
```

Comment:
Why did `const` disappear?

Reply:
The arguments passed to main() are not const, so there was no need for this const in the first place. I dropped it to make argv compatible with the prototype change in cmdline_main().